### PR TITLE
fix: UI_REACT: Create mapping when source field selected

### PR DESCRIPTION
**Pull request by jpav**
Fixes: #1903

### CODE REVIEW

File: ui-react/packages/atlasmap-core/src/services/mapping-management.service.ts [hunk 1]
```diff
@@ -425,9 +425,11 @@ export class MappingManagementService {
     if (selectedField != null) {
       mapping.addField(selectedField, false);
       this.updateTransition(mapping);
+      this.selectMapping(mapping);
       this.notifyMappingUpdated();
+    } else {
+      this.selectMapping(mapping);
```
Comment (pleacu) [inline ui-react/packages/atlasmap-core/src/services/mapping-management.service.ts:431]: Is this change really necessary?  Can't the selectMapping occur at the end?

Comment (jpav) [reply ui-react/packages/atlasmap-core/src/services/mapping-management.service.ts:431]: Yes, AFAICT, because the selectMapping call is what sets the activeMapping, which the subsequent notifyMappingUpdated call relies upon to update the list of mappings.
